docs/services/pixl_database.md

```diff
@@ -0,0 +1,7 @@
+# The PIXL database
+
+PIXL uses a [postgres database](../../postgres/README.md) to 
+
+- Add hashed identifiers along with the originals for DICOM images (in `pixl_dcmd`)
+- Keep track of the export status of images (in `core.uploader`)
```

Comment:
I'd probably say imaging studies and the projects that they are used in

Might be worth saying that the pipeline will not process any studies for a project that have already been exported?